packages/transmute-cli/src/commands/generate-keys/gpg_import

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+: ${SECRETS_DIR:=$HOME/.transmute/cli-secrets}
```

Comment:
So is the idea that you can move keys into this directory when importing them?

maybe add a short comment to the top of this script, with a reference to the wiki.